automation/ip-address-script/template.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "ipName": {
+            "defaultValue": "nsc-rg-dev-usw2-ip",
+            "type": "String"
+        },
+        "serverLocation": {
+            "defaultValue": "westus2",
```

Comment:
You could add here AllowedValues to limit the input possibilities

Reply:
Good idea!